applications/tari_console_wallet/src/grpc/wallet_grpc_server.rs

```diff
@@ -175,6 +197,53 @@ impl wallet_server::Wallet for WalletGrpcServer {
 
         Ok(Response::new(GetTransactionInfoResponse { transactions }))
     }
+
+    async fn get_all_completed_transactions(
+        &self,
+        _request: Request<GetAllCompletedTransactionsRequest>,
```

Comment:
This should either return a stream or use some paging of some kind. It's not hard to add

Reply:
Proposed as future change